### ISSUE TITLE
ScratchJr doesn't load in iOS 10.3 (beta)

### ISSUE THREAD

**Issue by chrisgarrity** [LLK/scratchjr#123]
We've now had a couple of reports from people running iOS10.3 beta where ScratchJr gets to the loading page (the index page with the two buttons), and then hangs. It's probably a permission issue, but we probably should try to test it before 10.3 is released officially.

Reported on:
iPad mini 2, on iOS 10.3 public beta 6
iPad Air2 latest 10.3 beta on Feb 15, 2017

**Comment by thisandagain** [comment]
@chrisgarrity This would probably be a good thing to get resolved before the next release. Let me know if you'd like any assistance! 😄

**Comment by chrisgarrity** [comment]
I think this issue is also related to the iOS sound bug #121 
I was able to reproduce the hanging behavior by upgrading an iPad to 10.3 and then downloading the current version from the App store. It hangs at the point on the launch screen where we're capturing touch events to activate the web audio context.

After reverting to 10.2, installing the latest version of ScratchJr with the sound bug fixed, and then updating to 10.3 the app no longer hangs and I was able to run programs, create and playback sounds etc. 

/cc @jwzimmer we probably need a plan for testing apps under new versions of iOS when the new version is still in beta. But for now I'm going to close this issue.
